### PR TITLE
Update node README.md

### PR DESCRIPTION
**Pull request by styfle**
This came up in https://github.com/nodejs/docker-node/issues/243#issuecomment-316411612

### CODE REVIEW

File: node/README.md [hunk 1]
```diff
@@ -93,44 +93,163 @@ $ docker run -it --rm --name my-running-app my-nodejs-app
...
+dependencies and defining its [start
+script](https://docs.npmjs.com/misc/scripts#default-values).
+
+It also assumes that you have a file named [`.dockerignore`](https://docs.docker.com/engine/reference/builder/#/dockerignore-file) otherwise it will copy your local npm modules:
+
+```
+node_modules
+```
+
+We have assembled a [Best Practices Guide](./docs/BestPractices.md) for those using these images on a daily basis.
```
Comment (SimenB) [inline node/README.md]: Will a relative link resolve correctly?

Comment (styfle) [inline node/README.md]: Good point! Probably not. I fixed this.

**Comment by tianon** [comment]
Please take a look at the CI failure -- this needs to be updated in `content.md` instead.  (Almost all the `README.md` files here are generated, as the comment at the top of each file notes.)

**Comment by tianon** [comment]
Also, we use a fork of `markdownfmt` (see `./markdownfmt.sh` in the root of this repo, and Travis's output) -- please follow the formatting it requires/suggests.

**Comment by SimenB** [comment]
If there are changes to the output after running `markdownfmt`, please PR those backe to node-docker as well 🙂

**Comment by styfle** [comment]
@tianon @SimenB I updated the PR so the `README.md` is unchanged.
Hopefully this works as expected.

dog

**Comment by styfle** [comment]
> The command ".travis/check-markdownfmt.sh" exited with 1.

What's wrong with the format this time?

**Comment by SimenB** [comment]
Can you run it locally?

**Comment by styfle** [comment]
@SimenB No, it doesn't seem to work with git bash on windows.

```
styfle@WINDOWS-PC MINGW64 /d/code/docker-docs (patch-1)
$ .travis/check-markdownfmt.sh
xargs: markdownfmt: No such file or directory
```

**Comment by SimenB** [comment]
Doing `go get -u github.com/shurcooL/markdownfmt` and `~/go/bin/markdownfmt -d node/license.md` and `~/go/bin/markdownfmt -d node/content.md` exits with `0`, so I don't know what's wrong...

**Comment by SimenB** [comment]
I also tried `./markdownfmt.sh node/*.md` (which uses a docker image) with no change

**Pull request merged**